javascript/javascript1/week3/homework2/script.js

```diff
@@ -0,0 +1,46 @@
+
+// SERIE DURATION
+
+const seriesDurations = [
+    {
+      title: "Breaking Bad",
+      days: 2,
+      hours: 3,
+      minutes: 40,
+    }
+  ];
+
+// CALCULATING TIME OF SERIES IN HOURS
+
+
+  const totalHoras = (seriesDurations[0].days * 24) + seriesDurations[0].hours + (seriesDurations[0].minutes / 60);
```

Comment:
We need to make this function and pass the other TV-series objects there as well